SDK/simVis/CMakeLists.txt

```diff
@@ -181,6 +182,7 @@ set(VIS_HEADERS_GOG
     ${VIS_INC}GOG/Ellipse.h
     ${VIS_INC}GOG/Ellipsoid.h
     ${VIS_INC}GOG/ErrorHandler.h
+    ${VIS_INC}GOG/GenericGeometry.h
```

Comment:
This and the .cpp look like a dead file.  I will remove them on a future pass.

Reply:
I wasn't sure exactly what it did, but it did appear to have been committed like that so I thought maybe there was a side effect somewhere that I was missing.